### ISSUE TITLE
Retrieve the Version number of the ChromaSDK

### ISSUE THREAD

**Issue by njbmartin** [CoraleStudios/Colore#159]
Useful for ensuring compatibility within apps. Some people refuse to update Synapse, not helped by the fact the release notes are unhelpful.

For example, if the version is lower than 1.4.2, some effects might not work.

**Comment by Sharparam** [comment]
Version information can be found in the following registry location: `HKLM\SOFTWARE\Razer Chroma SDK` from the keys `MajorVersion` (`DWORD`), `MinorVersion` (`DWORD`), and `RevisionNumber` (`DWORD`).

I've been thinking about adding constants in Colore that specifies which SDK version it's built for and issue a warning if the versions don't match.

**Comment by njbmartin** [comment]
I've used this in the Kraken Repair Tool:
```
string path = Environment.ExpandEnvironmentVariables(@"%PROGRAMFILES%\Razer Chroma SDK\bin\RzChromaSDK.dll");
FileVersionInfo fvi = FileVersionInfo.GetVersionInfo(path);
```

**Comment by Sharparam** [comment]
Closing this as the functionality was implemented in #161. Ability to also query file version might be useful but at this time it doesn't seem the file version is kept up to date? If new info arrives please re-open this issue.
